content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-Cosign supports container signing, verification, and storage in an OCI registry.
-Cosign aims to make signatures invisible infrastructure.
-
-![Cosign demo gif](/cosign.gif)
-
-Cosign supports:
+## Getting Started (Quick Start)
 
-<list :items="features" type="info"></list>
+Cosign supports blob and container signing, verification, and storage in an OCI registry.
+Cosign aims to make signatures invisible infrastructure.
 
 Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/) (need an [invite](https://links.sigstore.dev/slack-invite)?)
 
-## Getting Started (Quick Start)
-
-To get up and running we'll demonstrate how to: 
+### Installation
 
-1. Generate a keypair
-1. Sign a container image and store that signature in the registry
-1. Find signatures for a container image, and verify them against a public key
+To get sigstore up and running, you simply need to install Cosign. instructions to install Cosign can be found at https://docs.sigstore.dev/cosign/installation/. 
```

Comment:
Nit: is there agreement on whether the S should be capitalized in Sigstore?

Reply:
It used to be lower-case, but has been title case for about a year. [Linux Foundation](https://www.linuxfoundation.org/blog/blog/adopting-sigstore-incrementally) and [OpenSSF](https://openssf.org/community/sigstore/) both have it as title case